action-server/actions/action_qa_goodbye.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+
+CANCEL_CI_SLOT = "cancel_ci"
+
+
+class ActionQaGoodbye(Action):
+    def name(self) -> Text:
+        return "action_qa_goodbye"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+        if tracker.get_slot(CANCEL_CI_SLOT) is False:
+            dispatcher.utter_message(
+                template="utter_daily_ci__qa__will_contact_tomorrow"
+            )
```

Comment:
The name of the action is generic, yet this message is specific. By default, cancel_ci is False, although at the moment it might be set to None. Once its set to its default value, this might not work. I created an action_goodbye in my other PR, we should discuss this tomorrow.